Description: Fix FTBFS with gcc 4.7 by fixing missing <unistd.h> includes.
Author: Cyril Brulebois <kibi@debian.org>
Bug-Debian: http://bugs.debian.org/667108

--- bangarang-2.1.orig/src/platform/infofetchers/tmdbinfofetcher.cpp
+++ bangarang-2.1/src/platform/infofetchers/tmdbinfofetcher.cpp
@@ -30,6 +30,8 @@
 #include <QDomDocument>
 #include <QFile>
 
+#include <unistd.h>
+
 TMDBInfoFetcher::TMDBInfoFetcher(QObject *parent) :
         InfoFetcher(parent)
 {
--- bangarang-2.1.orig/src/platform/infofetchers/lastfminfofetcher.cpp
+++ bangarang-2.1/src/platform/infofetchers/lastfminfofetcher.cpp
@@ -32,6 +32,8 @@
 #include <QFile>
 #include <QTextEdit>
 
+#include <unistd.h>
+
 LastfmInfoFetcher::LastfmInfoFetcher(QObject *parent) :
         InfoFetcher(parent)
 {
--- bangarang-2.1.orig/src/platform/infofetchers/tvdbinfofetcher.cpp
+++ bangarang-2.1/src/platform/infofetchers/tvdbinfofetcher.cpp
@@ -30,6 +30,8 @@
 #include <QDomDocument>
 #include <QFile>
 
+#include <unistd.h>
+
 TVDBInfoFetcher::TVDBInfoFetcher(QObject *parent) :
         InfoFetcher(parent)
 {
